api/clamav_scanner/common.py

```diff
@@ -0,0 +1,48 @@
+import errno
+import datetime
+import os
+import os.path
+
+AV_DEFINITION_S3_BUCKET = os.getenv("AV_DEFINITION_S3_BUCKET")
+AV_DEFINITION_S3_PREFIX = os.getenv("AV_DEFINITION_S3_PREFIX", "clamav_defs")
+AV_DEFINITION_PATH = os.getenv("AV_DEFINITION_PATH", "/clamav/clamav_defs")
+AV_SCAN_START_SNS_ARN = os.getenv("AV_SCAN_START_SNS_ARN")
+AV_SCAN_START_METADATA = os.getenv("AV_SCAN_START_METADATA", "av-scan-start")
+AV_SIGNATURE_METADATA = os.getenv("AV_SIGNATURE_METADATA", "av-signature")
+AV_SIGNATURE_OK = "OK"
+AV_SIGNATURE_UNKNOWN = "UNKNOWN"
+AV_STATUS_CLEAN = os.getenv("AV_STATUS_CLEAN", "CLEAN")
+AV_STATUS_INFECTED = os.getenv("AV_STATUS_INFECTED", "INFECTED")
+AV_STATUS_METADATA = os.getenv("AV_STATUS_METADATA", "av-status")
```

Comment:
I'm going to update our scan Lambda to use this `av-` prefix and update the IAM policy on the bucket to prevent people from modifying those tag values from the console directly.